ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -112,12 +111,12 @@ def map_split(
         )
 
     def _map_sample(self, sample):
-        # using a Jinja template for some added flexibility, template can include variables and functions
         # e.g., {{ text }} or {{ text_proc.format_asr_text(text) }}
         try:
+            filtered_sample = {"sentence": sample["sentence"]}
```

Comment:
Hmm, this won't work. There are many other keys in the sample that some of the other templates make use of.

Maybe let's do an "exclude" list, with a regex like `.*audio.*` instead? We can also take the regex as an argument from the user instead of having it be fixed.

Reply:
Also it's best if you add a comment so that explains why the sample was filtered (otherwise someone might change it back later on).

Reply:
Think I might do an excludes fields list, because afraid we might accidently exclude a field we want with regex matching. Ie, a column might be called audio_transcript.